crates/mun/Cargo.toml

```diff
@@ -10,14 +10,16 @@ license = "MIT OR Apache-2.0"
 description = "Command-line interface for compiling, monitoring and running Mun code"
 
 [dependencies]
-failure = "0.1.7"
+anyhow = "1.0"
+thiserror = "1.0.17"
+displaydoc = "0.1"
```

Comment:
After creation of the original issue, a comment was [added](https://github.com/mun-lang/mun/issues/177#issuecomment-628609901) that proposed leaving out `displaydoc`. We agreed that that would be a better approach than using `displaydoc`. Maybe you missed that comment, but could you please remove the `displaydoc` usage?

This [article](https://nick.groenen.me/posts/rust-error-handling/) gives a really good explanation of how that can be achieved :)

Reply:
ok！